client/ayon_houdini/plugins/publish/extract_rop.py

```diff
@@ -29,7 +30,8 @@ def process(self, instance: pyblish.api.Instance):
         )
         ext = ext.lstrip(".")
 
-        self.render_rop(instance)
+        if creator_attribute.get("render_target", "local") == "local":
+            self.render_rop(instance)
```

Comment:
Maybe using the instance.data["farm"] for this condition?
```suggestion
        if not instance.data.get("farm", False):
            self.render_rop(instance)
```

Reply:
when instance doesn't have the `farm` instance data it'll either be: 
- local publish (do render)
- publish existing frames (skip render)

Reply:
> when instance doesn't have the `farm` instance data it'll either be:
> 
> * local publish (do render)
> * publish existing frames (skip render)

My bad, Been unaware of the other options of `publish existing frame`.
Take a look one more time, you can just do this.
```suggestion
        if creator_attribute.get("render_target") == "local":
            self.render_rop(instance)
```

Reply:
Some instances won't have `render_target` 
in that case, the fallback should be `local`. 

But, I'll check it again once I make other cache instances types have render targets.

Reply:
I checked the families in `ExtractRop`.
Unfortunately, There'll be some instances without the `render_target` key at the moment. instances with `imagesequence` specifically.

Reply:
let me add a comment in the a code about it.

Reply:
https://github.com/ynput/ayon-houdini/pull/180/commits/dbcce0b947f15ec002fc8b23031dd888ebbdb89b